app/controller/NavigationController.js

```diff
@@ -49,18 +49,18 @@ SDL.NavigationController = Em.Object.create(
      * @param {Object} request
      */
     sendLocation: function(request) {
-      this.model.LocationDetails.push(
+      this.model.LocationDetails.pushObject(
         {
           coordinate: {
             latitudeDegrees: request.params.latitudeDegrees,
-            longitudeDegrees: request.params.longitudeDegrees
+            longitudeDegrees: request.params.longitudeDegrees 

```

Comment:
@YarikMamykin please also remove this space

Reply:
@AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/7/commits/ccdf005934398ea67e5a62779fc851843b818e7c